app/src/main/java/com/keylesspalace/tusky/view/RoundedImageView.java

```diff
@@ -194,12 +194,10 @@ public ColorFilter getColorFilter() {
     }
 
     private void applyColorFilter() {
-        if (mBitmapPaint != null) {
-            mBitmapPaint.setColorFilter(mColorFilter);
-        }
+        mBitmapPaint.setColorFilter(mColorFilter);
```

Comment:
Didn't we.. add it for some reason? Can't remember

Reply:
It is a final variable thats always initialized, I really can't see any reason for the null check

Reply:
Ah sure